src/Core/Components/List/FluentAutocomplete.razor

```diff
@@ -204,7 +204,11 @@
                 {
                     var text = @GetOptionText(item);
 
-                    if (ReadOnly || Disabled)
+                    if (MaximumSelectedOptions == 1)
+                    {
+                        <FluentLabel aria-label="@GetOptionText(item)" role="checkbox" aria-checked="true">@text</FluentLabel>
```

Comment:
@dvoituron - I'm not sure if you think it appropriate to include a tab stop here or not. I left it out but could add it back in if you like.

Reply:
@dvoituron - I believe adding the tab stop is what you are looking for as far as accessibility goes.  I have added it so that you can see what the result is. Let me know if you still want other changes or if you still think we should pursue allowing `Multiple` = false;

Reply:
I'll check this out, but having a Multiple=false is still useful for people who want to keep the previous behavior.